### PR TITLE
Support `in` hook - #1

### PR DESCRIPTION
**Pull request by ivospinheiro**
Add support to set element attributes as Objects using **HTMLElement** **setAttribute**

### CODE REVIEW

File: react-to-webcomponent.js [hunk 1]
```diff
@@ -68,7 +72,16 @@ export default function(ReactComponent, React, ReactDOM) {
 		targetPrototype.attributeChangedCallback = function(name, oldValue, newValue) {
 			// TODO: handle type conversion
 			this[name] = newValue;
-		}
+		};
+
+		var setAttribute = targetPrototype.setAttribute;
+		targetPrototype.setAttribute = function (name, val) {
```
Comment (justinbmeyer) [inline react-to-webcomponent.js]: what is this for?

Comment (justinbmeyer) [reply react-to-webcomponent.js]: Why would you expect:

```
fixture.firstElementChild.setAttribute("person", { first: "Ivo", last: "Pinheiro" });
```

to work? If someone wants to pass data, they must set properties:

```js
fixture.firstElementChild.person = { first: "Ivo", last: "Pinheiro" }
```

Comment (ivospinheiro) [reply react-to-webcomponent.js]: I've done this change, after testing **react-to-webcomponent** together with **CanJS**.
I've noticed the **HTMLElement** **setAttribute** function was being called, and the native HTMLElement converts Element attribute value to string, that's why I had to override the implementation.

The code that was calling the **setAttribute** function is:
https://github.com/canjs/can-attribute-observable/blob/576d1b87acf1ac8d9fc7788ccb871c9fd1981a08/behaviors.js#L510

Maybe there is a better alternative.

Comment (justinbmeyer) [reply react-to-webcomponent.js]: Can you share your example code that was doing this?

Comment (ivospinheiro) [reply react-to-webcomponent.js]: Here is the code:
https://gist.github.com/ivospinheiro/b85f053a307704fc8f98744fc9e7ad20

**Pull request merged**